docs/alma/alma.rst

```diff
@@ -181,21 +181,23 @@ or if you wanted all projects by a given PI:
 Querying For Proprietary Data
 -----------------------------
 
-To query for proprietary data (non-public data), you need to first authenticate using the ``login`` method,
-and then set ``public=False`` in your query:
+To query for proprietary (non-public) data, set ``public=False`` or ``public=None``.
+If you want to download the products, you need to first authenticate using the ``login`` method.
 
 .. doctest-skip::
 
     >>> from astroquery.alma import Alma
     >>> alma = Alma()
     >>> # First login to access proprietary data
     >>> alma.login("username")  # Will prompt for password
-    >>> # Now query including proprietary data
-    >>> proprietary_data = alma.query(payload=dict(project_code='2017.1.01355.L', public=False))
-    >>> # Or restrict to only proprietary data
+    >>> # include both public and proprietary data using public=None
+    >>> proprietary_data = alma.query(payload=dict(project_code='2017.1.01355.L', public=None))
```

Comment:
I forgot what happens for `public=None` and unauthenticated user. Returns only public data or fails?